source/docs/software/frc-glossary.rst

```diff
@@ -11,21 +11,51 @@ FRC Glossary
    auto
       The first phase of each match is called Autonomous (auto) and consists of the robot's running pre-programmed instructions.
 
+   call stack
+      A specially-organized region of memory which helps the program keep track of what function it is in, and store the values of some variables. See `call stack <https://en.wikipedia.org/wiki/Call_stack>`__ on wikipedia for more info.
```

Comment:
"Since code functions call other functions in a nested manner, the current location of code generally appears as a "stack" of nested function calls, hence the term, "call stack.""

Reply:
I got slightly more liberal with the wording, but added a reference to this "stuff gets put on top of other stuff = stack"